catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -44,15 +44,13 @@
 
     private Set<String> registryEntryIds = new HashSet<>();
 
-    private List<String> registryBypassPolicyStrings;
-
-    private List<String> writeAccessPolicyStrings;
```

Comment:
I think we are safe with just Maps here. All the other implementations of the Policy Plugin also use Maps/Lists.